.docker/opt/configure-aiida.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# This script is executed whenever the docker container is (re)started.
+
+# Debugging.
+#set -x
```

Comment:
I've often found it useful to have this on.
Perhaps let's start with `set -x` and if in a few months everything is well converged, we can deactivate it again.